### ISSUE TITLE
Change Language Label for Cantonese and Mandarin

### ISSUE THREAD

**Issue by nessmak** [lookit/lookit-api#1135]
# TL;DR
Make clearer the language exposure choice for Mandarin and Cantonese by changing the labels for ISO codes "cmn" and "yue" 

# Narrative
As a parent registering my Chinese-English bilingual child, I am currently having trouble deciphering between the labels for all the Chinese languages in the section "Languages this child is exposed to at home, school, or with another caregiver". I would like to see clearly, an option for "Mandarin" and an option for "Cantonese. 

# Acceptance Criteria

- [ ] ISO code "cmn" is labelled "Chinese (Mandarin)"
- [ ] ISO code "yue" is labelled "Chinese (Cantonese)"

# Implementation Notes
<img width="686" alt="Screen Shot 2023-03-10 at 11 07 48 PM" src="https://user-images.githubusercontent.com/71201668/224438809-6472c178-0bc6-4fb5-b4ea-08922f389631.png">
